src/sentry/api/bases/organizationmember.py

```diff
@@ -1,21 +1,48 @@
 from django.db.models import Q
+from rest_framework import serializers
 
 from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.db.models.fields.bounded import BoundedAutoField
 from sentry.models import OrganizationMember
 
 from .organization import OrganizationEndpoint
 
 
+class MemberIdField(serializers.IntegerField):
```

Comment:
@wedamija is this is a sensible way to do this validation -- we want "me" to be allowed in addition to other integer values for validating member ids.

Reply:
I would try to make this a broader field that applies an explicit transformation, such as
```python
return data if data == "me" else int(data)
```
relying on the `ValueError` raised by `int(data)` if it's not a valid string.